tests/unit/customizations/test_opsworks.py

```diff
@@ -734,6 +734,23 @@ def test_validate_same_region(self, mock_urlopen):
             self.register.validate_arguments(
                 self._build_args(hostname=None, local=True))
 
+    @mock.patch.object(opsworks, "urlopen")
+    def test_validate_same_region_bytes(self, mock_urlopen):
+        """Same as the above test, on some OSes urlopen.read returns a bytes"""
+
+        self.register._stack = {"Region": "mars-east-1"}
+
+        mock_urlopen.return_value.read.return_value = \
+            b'{"region": "mars-east-1"}'
+        self.register.validate_arguments(
+            self._build_args(hostname=None, local=True))
+
+        with self.assertRaises(ValueError):
```

Comment:
I copied the above test so it still does the same test just with bytes instead of strings so that ValueError is checking for east/west. I suppose I could cut out the second portion of the test since it is irrelevant.